lib/data_cycle/daily_update.rb

```diff
@@ -33,6 +34,7 @@ def run_update
     update_users
     update_commons_uploads
     update_article_data
+    update_wiki_discouraged_article
```

Comment:
I guess this should also use `if Features.wiki_ed?`, as we only anticipate using this feature on Wiki Education Dashboard.